gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineErrorProne.java

```diff
@@ -154,6 +154,17 @@ private static void applyToJavaProject(Project project) {
                                 errorProneOptions.check("PreferSafeLoggingPreconditions", CheckSeverity.OFF);
                                 errorProneOptions.check("PreconditionsConstantMessage", CheckSeverity.OFF);
                             }));
+
+            project.getTasks().withType(JavaCompile.class).configureEach(javaCompile -> {
```

Comment:
is this intentionally only disabled for people that use `java-gradle-plugin`? It feels a little odd to disable it in such a specific case.

If we think this UnnecessaryLambda is genuinely not great signal, why not just turn it off for all test code?

Reply:
that's what i meant to do, sorry